modules/driver-mongo/build.gradle.kts

```diff
@@ -1,30 +1,13 @@
 plugins {
-    id("org.jetbrains.kotlin.jvm")
-    `maven-publish` // Needed for Jitpack
-    id("com.adarshr.test-logger")
-}
-
-baseProject()
-
-publishing {
-    publications {
-        create<MavenPublication>("maven") {
-            groupId = project.group.toString()
-            artifactId = "datamaintain-" + project.name
-            version = project.version.toString()
-
-            from(components["java"])
-        }
-    }
+    id("datamaintain.conventions.kotlin")
+    id("datamaintain.conventions.publishing")
+    id("datamaintain.conventions.driver")
 }
 
 dependencies {
-    compileOnly(project(":modules:core"))
-    testImplementation(project(":modules:core"))
-
-    testImplementation("org.mongodb:mongodb-driver-sync:${Versions.mongoDriver}")
+    testImplementation(libs.mongoDriver.sync)
 
     // Use kotlinx-serialization implementation
-    testImplementation(project(":modules:driver-mongo-mapping:driver-mongo-mapping-serialization"))
-    testImplementation("org.jetbrains.kotlinx:kotlinx-serialization-json:${Versions.kotlinxSerialization}")
+    testImplementation(projects.modules.driverMongoMapping.driverMongoMappingSerialization)
```

Comment:
There's actually no use for that dependency, let's remove it

Reply:
Done